snuba/datasets/configuration/entity_builder.py

```diff
@@ -152,6 +152,35 @@ def _build_storage_connections(
     ]
 
 
+def _build_join_relationships(config: dict[str, Any]) -> dict[str, JoinRelationship]:
+    relationships: dict[str, JoinRelationship] = {}
+    if "join_relationships" not in config:
+        return relationships
+
+    for key, obj in config["join_relationships"].items():
+        rhs_key = register_entity_key(obj["rhs_entity"])
+        columns = [(c[0], c[1]) for c in obj["columns"]]
+        equivalences = []
+        for pair in obj.get("equivalences", []):
+            equivalences.append(ColumnEquivalence(pair[0], pair[1]))
+
+        if obj["join_type"] not in ("inner", "left"):
+            raise InvalidEntityConfigException(
+                f"{obj['join_type']} is not a valid join type"
+            )
+
+        join_type = JoinType.LEFT if obj["join_type"] == "left" else JoinType.INNER
+        join = JoinRelationship(
+            rhs_entity=rhs_key,
+            columns=columns,
+            join_type=join_type,
+            equivalences=equivalences,
+        )
+        relationships[key] = join
+
+    return relationships
+
+
```

Comment:
Did you change this in your PR or is this merge weirdness?

Reply:
Definitely merge weirdness, that was from a PR I merged.

Reply:
Yup, merge weirdness. Updated.